parastell/nwl_utils.py

```diff
@@ -186,6 +186,9 @@ def flux_coords(plas_eq, wall_s, coords, num_threads):
             for theta_coord_chunk in theta_coord_chunks
             for theta_coord in theta_coord_chunk
         ]
+    # Ensures theta_coords are all positive (add 360 degrees where needed)
+    theta_coords = (theta_coords + 2 * np.pi) % 2*np.pi
```

Comment:
will they need to be reordered/sorted?

Reply:
Doesn't need to be reordered, theta_coords is just a list of the same length as the list of surface crossings from the surface source that stores the poloidal angle for each surface crossing, all we are doing here is changing the notation of theta, each theta_coord still corresponds to the same surface crossing.

For reference, here is what the NWL looks like from before:
![image](https://github.com/user-attachments/assets/70e552e4-d275-4156-a12c-f8185497687a)
And after:
![image](https://github.com/user-attachments/assets/4345bb7d-6b54-47a0-9a68-b381d5041591)

The first has been rescaled as noted in its title, but looking at the overall profile the coordinate shift can be seen